examples/browser/package.json

```diff
@@ -10,7 +10,7 @@
     "monaco-languages": "^0.7.0",
     "monaco-typescript": "^2.1.2",
     "reflect-metadata": "^0.1.10",
-    "theia": "file:../../../Theia"
+    "theia": "file:../../../theia"
```

Comment:
Good catch, but now it will break for those who have the repo already checked out.  I have tried to use just `file:../..` and it seems to work, I think we should use that instead.